### PR TITLE
Tutorial: transfer-files (previsously #714)

### PR DESCRIPTION
**Pull request by dignifiedquire**
Ref #714

### CODE REVIEW

File: test/cli/bitswap.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe('bitswap', () => runOn((thing) => {
     ipfs('block get ' + key)
       .then(() => {})
       .catch(() => {})
-    setTimeout(done, 100)
+    setTimeout(done, 800)
```
Comment (diasdavid) [inline test/cli/bitswap.js]: why the need to increase the timeout?

Comment (dignifiedquire) [inline test/cli/bitswap.js]: because it randomly fails on CI due to things being slow

**Comment by dignifiedquire** [comment]
@diasdavid https://github.com/ipfs/js-ipfs/pull/772 but needs updating probably

**Comment by diasdavid** [comment]
@dignifiedquire got it, could you remove the yarn.lock from here then?

**Comment by VictorBjelkholm** [comment]
Worth to note is that this is **not** ready to be merged, the document at `examples/transfer-files/README.md` is still unfinished.

**Comment by diasdavid** [comment]
Thanks for bringing that up @VictorBjelkholm 

# TODO

- [ ] Peers list have a comma between every peer that is not supposed to be there
- [ ] Multiaddr input is too wide and goes outside the containing box
- [ ] Refactor code + add valuable comments
- [x] Have .get actually work with just one file 
- [ ] Fix drag-and-drop not closing popup when ending up not dropping file in UI
- [ ] Finish tutorial in readme.md

**Comment by VictorBjelkholm** [comment]
>  Multiaddr input is too wide and goes outside the containing box

Should not happen anymore, and I remember when Friedel demod multiplex by using this demo, it didn't happen anymore. But I'll verify to make sure. 

Seems otherwise to capture everything that is missing, thanks for that 👍

**Pull request merged**